### PR TITLE
docs: update changelog manually for 8.19

### PR DESCRIPTION
**Pull request by frodehk**
ECALC-1428

## Why is this pull request needed?

The changelog for 8.19 includes changes from several releases back, not only changes related to 8.19.

## What does this pull request change?

Ensure that only changes from 8.19 are included in the changelog for 8.19.

## Issues related to this change:
https://equinor-ecalc.atlassian.net/browse/ECALC-1428?atlOrigin=eyJpIjoiZjAxYzM1NGNkNDFhNGM5N2E4OGIxYzdhZGVmMWFmMDMiLCJwIjoiaiJ9

### CODE REVIEW

File: docs/docs/changelog/changelog.md [hunk 1]
```diff
@@ -15,557 +15,31 @@
...
-* new PfS columns ([#429](https://github.com/equinor/ecalc/issues/429)) ([a6b198e](https://github.com/equinor/ecalc/commit/a6b198efa45a6caf7464d2d1dc0d2509d5d76541))
-* output pump head to json-file ([#49](https://github.com/equinor/ecalc/issues/49)) ([60720f4](https://github.com/equinor/ecalc/commit/60720f429cb5da82cd839740eca8d3039c9d5969))
-* support bfs_tree in graph ([a4fff16](https://github.com/equinor/ecalc/commit/a4fff168dfa32d04588bb7e3b71de8c12e9dd6d0))
-* support name for crossover streams ([#236](https://github.com/equinor/ecalc/issues/236)) ([c801f3f](https://github.com/equinor/ecalc/commit/c801f3f0fa4b967c15c5122ac8997695f38bae12))
-* train v2 yaml and dto ([#272](https://github.com/equinor/ecalc/issues/272)) ([b0e3466](https://github.com/equinor/ecalc/commit/b0e346618bb3b39a186eb814dd40be2f7d905122))
-* validate expression reference tokens ([#404](https://github.com/equinor/ecalc/issues/404)) ([2af9d13](https://github.com/equinor/ecalc/commit/2af9d1332864b3e3bf17f84bd980b8bd436e7c7a))
-* validate model references and types ([dd8d701](https://github.com/equinor/ecalc/commit/dd8d701a5ac546407e6dcd0549991c3c73dff106))
```
Comment (TeeeJay) [inline docs/docs/changelog/changelog.md:52]: this should be included, line 52

**Review by TeeeJay — APPROVED** [review]
good, just one missed :)

File: docs/docs/changelog/changelog.md (hunk 1 above)
Comment (frodehk) [reply docs/docs/changelog/changelog.md:52]: It is included in 8.18.1, should it still be in 8.19?

Comment (TeeeJay) [reply docs/docs/changelog/changelog.md:52]: aha, diff mismatch then, no then it should not :)

**Pull request merged**